src/system.c

```diff
@@ -825,7 +832,10 @@ void sysLaunchLoaderElf(const char *filename, const char *mode_str, int size_cdv
 #ifdef __DECI2_DEBUG
     modules |= CORE_IRX_DECI2 | CORE_IRX_ETH;
 #elif defined(__INGAME_DEBUG)
-    modules |= CORE_IRX_DEBUG | CORE_IRX_ETH;
+    modules |= CORE_IRX_DEBUG;
+#ifdef TTY_UDP
+    modules |= CORE_IRX_ETH;
+#endif
```

Comment:
doh it seems you cannot combine it in one. What about if I select both `__INGAME_DEBUG` and `TTY_UDP` ? Logic before this PR is also questionable, but this change will effectively break udp debugging ingame

Reply:
I'm open to suggestions

Reply:
Ah, yes my bad, didnt realize that this is another ifdef not elif. Maybe ident it.

Reply:
> Ah, yes my bad, didnt realize that this is another ifdef not elif. Maybe ident it.

IIRC current clang format config hates macro indent

Reply:
nah its not necessary. Changes approved.